zproject/sentry.py

```diff
@@ -32,6 +32,11 @@ def add_context(event: 'Event', hint: 'Hint') -> Optional['Event']:
                 # str() to force the lazy-translation to apply now,
                 # since it won't serialize into json for Sentry otherwise
                 user_info["role"] = str(user_profile.get_role_name())
+        # These are PII, and should be scrubbed
+        if "username" in user_info:
+            del user_info["username"]
+        if "email" in user_info:
+            del user_info["email"]
```

Comment:
Should `delivery_email` be in here?  

Also, when do we use `username`?

Reply:
Ah, no.  [`user_info` is the Sentry user information object](https://docs.sentry.io/platforms/python/guides/django/enriching-error-data/additional-data/identify-user/), which has been pre-populated by the Django integration.  I'll add a comment to clarify.